client/src/components/layout/ReviewTile.js

```diff
@@ -1,10 +1,12 @@
 import React from "react"
 
-const ReviewTile = ({ id, rating, comment }) => {
+const ReviewTile = (props) => {
+    const currentReview = props.review
```

Comment:
if you use destructuring in other places, you should continue to do so here

either: `const ReviewTile = ({ review }) => {`

or: 
```
const ReviewTile = (props) => {
    const { review } = props
```